Testscripts/Linux/dpdk_verify_primary_secondary.sh

```diff
@@ -47,17 +47,17 @@ function build_test_dpdk_primary_secondary () {
 	done
 
 	LogMsg "Whitelisted PCI ids: ${whitelist_params}"
+	mp_server_log_file="./primary_secondary_server.log"
+	LogMsg "${RTE_SDK}/${EXAMPLES_DIR}/mp_server/${RTE_TARGET}/mp_server -l0-1 -n4 $whitelist_params -- -p 0x14 -n2  2>&1 > $mp_server_log_file &"
+	"${RTE_SDK}/${EXAMPLES_DIR}/mp_server/${RTE_TARGET}/mp_server" -l0-1 -n4 $whitelist_params -- -p 0x14 -n2  2>&1 > $mp_server_log_file &
+	sleep 30
+	mp_client_log_file="./primary_secondary.log"
+	LogMsg "timeout --preserve-status 30 ${RTE_SDK}/${EXAMPLES_DIR}/mp_client/${RTE_TARGET}/mp_client -l3 \
+		-n4 --proc-type=auto $whitelist_params -- -n 0 2>&1 > $mp_client_log_file"
+	timeout --preserve-status 30 "${RTE_SDK}/${EXAMPLES_DIR}/mp_client/${RTE_TARGET}/mp_client" -l3 \
+		-n4 --proc-type=auto $whitelist_params -- -n 0 2>&1 > $mp_client_log_file
 
-	LogMsg "${RTE_SDK}/${EXAMPLES_DIR}/mp_server/${RTE_TARGET}/mp_server -l0-1 -n4 $whitelist_params -- -p 0x14 -n2 1>/dev/null 2>/dev/null &"
-	"${RTE_SDK}/${EXAMPLES_DIR}/mp_server/${RTE_TARGET}/mp_server" -l0-1 -n4 $whitelist_params -- -p 0x14 -n2 1>/dev/null 2>/dev/null &
-	sleep 10
-	log_file="./primary_secondary.log"
-	LogMsg "timeout --preserve-status 15 ${RTE_SDK}/${EXAMPLES_DIR}/mp_client/${RTE_TARGET}/mp_client -l3 \
-		-n4 --proc-type=auto $whitelist_params -- -n 0 2>&1 > $log_file"
-	timeout --preserve-status 15 "${RTE_SDK}/${EXAMPLES_DIR}/mp_client/${RTE_TARGET}/mp_client" -l3 \
-		-n4 --proc-type=auto $whitelist_params -- -n 0 2>&1 > $log_file
-
-	test_output=$(cat $log_file)
+	test_output=$(cat $mp_client_log_file)
```

Comment:
You are handling client log file here, but did not do with server log file. Are they identical or do you have any other reason?

Reply:
I want to see logs from both server and client, though client log can be used for judge case fail or not.